test/preprocess_tests/filter_molecule_table_test.py

```diff
@@ -7,7 +7,7 @@
 from cassiopeia.preprocess import pipeline
 
 
-class TestFilterMoleculeTable(unittest.TestCase):
+class TestFiltermolecule_table(unittest.TestCase):
```

Comment:
Not a big deal, but normally class names are camel case in Python (at least in our codebase) so we can keep the old nomenclature.

Reply:
Whoops haha. It must have gotten replaced when I was replacing everything 😢